sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The CopyAuthorization model.
+ */
+@Immutable
+public final class CopyAuthorization {
+
+    /*
+     * Model identifier.
+     */
+    private final String modelId;
+
+    /*
+     * Token claim used to authorize the request.
+     */
+    private final String accessToken;
+
+    /*
+     * Resource Identifier.
+     */
+    private final String resourceId;
+
+    /*
+     * Region of the resource.
+     */
+    private final String resourceRegion;
+
+    /*
+     * The time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     */
+    private final long expirationDateTimeTicks;
+
+    /**
+     * Create a CopyAuthorization object
+     *
+     * @param modelId The model identifier
+     * @param accessToken The token used to authorize the request
+     * @param resourceId The resource identifier
+     * @param resourceRegion The region of the resource
+     * @param expirationDateTimeTicks The expiry time of the token
+     */
+    public CopyAuthorization(final String modelId, final String accessToken, final String resourceId,
+        final String resourceRegion,
+        final long expirationDateTimeTicks) {
+        this.modelId = modelId;
+        this.accessToken = accessToken;
+        this.resourceId = resourceId;
+        this.resourceRegion = resourceRegion;
+        this.expirationDateTimeTicks = expirationDateTimeTicks;
+    }
+
+    /**
+     * Get the modelId property.
+     *
+     * @return the modelId value.
+     */
+    public String getModelId() {
+        return this.modelId;
+    }
+
+    /**
+     * Get the token claim used to authorize the request.
+     *
+     * @return the accessToken value.
+     */
+    public String getAccessToken() {
+        return this.accessToken;
+    }
+
+    /**
+     * Get the time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     *
+     * @return the expirationDateTimeTicks value.
+     */
+    public long getExpirationDateTimeTicks() {
```

Comment:
in .NET we are exposing the date itself and not the ticks. Consider changing the format too so it is more useful for the user.

Reply:
Also, the name we use is `ExpiresOn` see https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyAuthorization.cs#L14

Reply:
Updated